client/my-sites/earn/memberships/index.jsx

```diff
@@ -480,16 +535,23 @@ class MembershipsSection extends Component {
 	}
 
 	renderOnboarding( cta ) {
+		const isPayments = isEnabled( 'earn/rename-payment-blocks' );
 		return (
 			<div className="memberships__onboarding-wrapper">
 				<div className="memberships__onboarding-column-info">
 					<div className="memberships__onboarding-header">
-						{ this.props.translate( 'Introducing Recurring Payments.' ) }
+						{ isPayments
+							? this.props.translate( 'Introducing Payments.' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Introducing… Simple Payments' )` *ES Score: 7*